WordPress/Tests/Functions/ReturnTypeUnitTest.inc.fixed

```diff
@@ -0,0 +1,80 @@
+<?php
+
+function fooa(): array {} // Good.
+function foob(): array {} // Bad.
+function fooc(): array {} // Bad.
+function food(): array {}
```

Comment:
No, I took the comment out of the .fixed file to make the tests pass for now :-)